conf/portal.application.conf

```diff
@@ -130,6 +130,10 @@ hostProvider {
 # Rollups
 # ~~~~~
 rollup {
+  # TTL on the individual rollup series.
+  # Specifying 0 will use the global TTL configured by KairosDB.
```

Comment:
are we 100% sure of this? If so, accepted.

Reply:
Based on https://kairosdb.github.io/docs/build/html/restapi/Aggregators.html#save-as

I suppose that could be completely wrong

Reply:
Did my due-diligence, in the fork we run it defaults to 0 anyway when unset:

https://github.com/InscopeMetrics/kairosdb/blob/master/src/main/java/org/kairosdb/events/DataPointEvent.java#L36

constructed via

https://github.com/InscopeMetrics/kairosdb/blob/master/src/main/java/org/kairosdb/core/aggregator/SaveAsAggregator.java#L172